sass/px-modal.scss

```diff
@@ -20,14 +20,8 @@
 // Objects
```

Comment:
`px-flexbox-design` is imported but isn't used.

Reply:
Fixed